plugins/woocommerce/client/admin/client/settings-payments/components/country-selector/utils.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { Item } from './types';
+
+// Get the label for the selected option.
+export const getOptionLabel = ( value: string, options: Item[] ) => {
```

Comment:
Do we really need this very simple utility function? It seems like we can directly use its one-liner statement in the calling code.

Reply:
Great point! I initially created the additional file because there were a few functions, but now it doesn’t seem necessary. I’ve removed it.